.gitignore

```diff
@@ -10,9 +10,7 @@ lib64
 pyvenv.cfg
 
 # Test files or temporary data files
-*.TextGrid
 *.pfsx
-*.eaf
```

Comment:
When using `pytest`'s `tmp_path` fixture, we can make sure that temporary files are not written to the repository.